### PR TITLE
Adding readiness

### PR DESCRIPTION
**Pull request by bardielle**
@nirarg pay attention that you (may) need to change the api in the installer

### CODE REVIEW

File: cmd/manager/main.go [hunk 1]
```diff
@@ -144,6 +145,14 @@ func main() {
...
 	}
 
+	if err := mgr.AddReadyzCheck("ping", healthz.Ping); err != nil {
+		klog.Fatal(err)
+	}
+
+	if err := mgr.AddHealthzCheck("ping", healthz.Ping); err != nil {
+		klog.Fatal(err)
+	}
+
```
Comment (chenyosef) [inline cmd/manager/main.go:155]: I really don't think it will fail if it didn't on the first if (-: - remove redundant code

**Review by ravidbro — APPROVED** [review]

File: pkg/managers/vm/machine_scope.go [hunk 2]
```diff
@@ -32,7 +32,7 @@ const (
 const (
 	defaultRequestedMemory            = "2048M"
 	defaultRequestedStorage           = "35Gi"
-	defaultPersistentVolumeAccessMode = corev1.ReadWriteOnce
+	defaultPersistentVolumeAccessMode = corev1.ReadWriteMany
```
Comment (bardielle) [inline pkg/managers/vm/machine_scope.go:35]: @nirarg @ravidbro Please pay attention - I changed the default value to be ReadWriteMany and in libvirt the PVC are can me only in ReadWriteOnce mode

Comment (ravidbro) [reply pkg/managers/vm/machine_scope.go:35]: Marius just changed it yesterday in the jenkins job, new cluster should have RWX PVs

Comment (bardielle) [reply pkg/managers/vm/machine_scope.go:35]: Great!

Comment (bardielle) [reply pkg/managers/vm/machine_scope.go:35]: @nirarg We need to change the source pvc to be also with the same access mode

**Pull request merged**